docs/en/observability/tab-widgets/uptime-monitoring/set-up-connect-content.asciidoc

```diff
@@ -0,0 +1,74 @@
+// tag::heartbeat[]
+
+Before we proceed, you'll need to retrieve your Elasticsearch credentials for either an {heartbeat-ref}/configure-cloud-id.html[Elastic Cloud ID] or another {heartbeat-ref}/elasticsearch-output.html[Elasticsearch Cluster].
+
+WARNING: Elastic synthetics runs Chromium without the extra protection of its process https://chromium.googlesource.com/chromium/src/+/master/docs/linux/sandboxing.md[sandbox] for greater compatibility with Linux server distributions.
```

Comment:
Can we make this a footnote or similarly shove it out of the way? This is the sort of thing we should document for completeness, but isn't a practical consideration for most.

Reply:
++ ("just in time, not just in case")